Index: TicTac.iml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n<module external.linked.project.id=\"TicTac\" external.linked.project.path=\"$MODULE_DIR$/TicTac\" external.root.project.path=\"$MODULE_DIR$/TicTac\" external.system.id=\"GRADLE\" external.system.module.group=\"org.example\" external.system.module.version=\"1.0-SNAPSHOT\" type=\"JAVA_MODULE\" version=\"4\">\r\n  <component name=\"NewModuleRootManager\" inherit-compiler-output=\"true\">\r\n    <exclude-output />\r\n    <content url=\"file://$MODULE_DIR$/TicTac\">\r\n      <excludeFolder url=\"file://$MODULE_DIR$/TicTac/.gradle\" />\r\n      <excludeFolder url=\"file://$MODULE_DIR$/TicTac/build\" />\r\n    </content>\r\n    <orderEntry type=\"jdk\" jdkName=\"22\" jdkType=\"JavaSDK\" />\r\n    <orderEntry type=\"sourceFolder\" forTests=\"false\" />\r\n  </component>\r\n</module>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/TicTac.iml b/TicTac.iml
--- a/TicTac.iml	(revision a9264d3c78b7541afd32d982d27825487b05e215)
+++ b/TicTac.iml	(date 1748783609382)
@@ -6,7 +6,7 @@
       <excludeFolder url="file://$MODULE_DIR$/TicTac/.gradle" />
       <excludeFolder url="file://$MODULE_DIR$/TicTac/build" />
     </content>
-    <orderEntry type="jdk" jdkName="22" jdkType="JavaSDK" />
+    <orderEntry type="inheritedJdk" />
     <orderEntry type="sourceFolder" forTests="false" />
   </component>
 </module>
\ No newline at end of file
Index: .idea/modules.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n<project version=\"4\">\r\n  <component name=\"ProjectModuleManager\">\r\n    <modules>\r\n      <module fileurl=\"file://$PROJECT_DIR$/.idea/SLT-TikTakToe.iml\" filepath=\"$PROJECT_DIR$/.idea/SLT-TikTakToe.iml\" />\r\n      <module fileurl=\"file://$PROJECT_DIR$/TicTac.iml\" filepath=\"$PROJECT_DIR$/TicTac.iml\" />\r\n      <module fileurl=\"file://$PROJECT_DIR$/.idea/modules/TicTac.main.iml\" filepath=\"$PROJECT_DIR$/.idea/modules/TicTac.main.iml\" />\r\n      <module fileurl=\"file://$PROJECT_DIR$/.idea/modules/TicTac.test.iml\" filepath=\"$PROJECT_DIR$/.idea/modules/TicTac.test.iml\" />\r\n    </modules>\r\n  </component>\r\n</project>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/modules.xml b/.idea/modules.xml
--- a/.idea/modules.xml	(revision a9264d3c78b7541afd32d982d27825487b05e215)
+++ b/.idea/modules.xml	(date 1748783609402)
@@ -6,6 +6,8 @@
       <module fileurl="file://$PROJECT_DIR$/TicTac.iml" filepath="$PROJECT_DIR$/TicTac.iml" />
       <module fileurl="file://$PROJECT_DIR$/.idea/modules/TicTac.main.iml" filepath="$PROJECT_DIR$/.idea/modules/TicTac.main.iml" />
       <module fileurl="file://$PROJECT_DIR$/.idea/modules/TicTac.test.iml" filepath="$PROJECT_DIR$/.idea/modules/TicTac.test.iml" />
+      <module fileurl="file://$PROJECT_DIR$/.idea/modules/org.example.TicTac.main.iml" filepath="$PROJECT_DIR$/.idea/modules/org.example.TicTac.main.iml" />
+      <module fileurl="file://$PROJECT_DIR$/.idea/modules/org.example.TicTac.test.iml" filepath="$PROJECT_DIR$/.idea/modules/org.example.TicTac.test.iml" />
     </modules>
   </component>
 </project>
\ No newline at end of file
Index: .idea/compiler.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n<project version=\"4\">\r\n  <component name=\"CompilerConfiguration\">\r\n    <bytecodeTargetLevel>\r\n      <module name=\"TicTac\" target=\"22\" />\r\n      <module name=\"TicTac.main\" target=\"22\" />\r\n      <module name=\"TicTac.test\" target=\"22\" />\r\n    </bytecodeTargetLevel>\r\n  </component>\r\n</project>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/compiler.xml b/.idea/compiler.xml
--- a/.idea/compiler.xml	(revision a9264d3c78b7541afd32d982d27825487b05e215)
+++ b/.idea/compiler.xml	(date 1748783578100)
@@ -2,6 +2,8 @@
 <project version="4">
   <component name="CompilerConfiguration">
     <bytecodeTargetLevel>
+      <module name="org.example.TicTac.main" target="22" />
+      <module name="org.example.TicTac.test" target="22" />
       <module name="TicTac" target="22" />
       <module name="TicTac.main" target="22" />
       <module name="TicTac.test" target="22" />
